pkg/vendir/fetch/hg/hg.go

```diff
@@ -99,11 +103,12 @@ func (t *Hg) fetch(dstPath string, tempArea ctlfetch.TempArea) error {
 			return fmt.Errorf("Parsing hg remote url: %s", err)
 		}
 
-		authRc = fmt.Sprintf(`[auth]
+		hgRc = fmt.Sprintf(`%s
```

Comment:
i think we should do hgRc += (and may be add a newline before) instead of overriding it entirely.

Reply:
In fact I need to string format other arguments, so overwriting it in this case use only one `fmt.Sprintf` call, otherwise it require one `fmt.Sprintf` call and the a `+=` call.

Let me know if you still want that.